### PR TITLE
Remove getRank

### PR DESCRIPTION
**Pull request by nishadg246**
Before adding distance sorting, the sorting was based on a rank which
was always a value of .5.

### CODE REVIEW

File: java/libs/src/main/java/org/physical_web/collection/PhysicalWebCollection.java [hunk 1]
```diff
@@ -324,7 +324,7 @@ public PwPair getTopRankedPwPairByGroupId(String groupId, Comparator<PwPair> com
         // Create the group if it doesn't exist
         UrlGroup urlGroup = groupMap.get(groupId);
         if (urlGroup == null) {
-          urlGroup = new UrlGroup(groupId);
+          urlGroup = new UrlGroup(groupId, comparator);
```
Comment (cco3) [inline java/libs/src/main/java/org/physical_web/collection/PhysicalWebCollection.java]: I don't think the group itself should know about the comparator.  Just pass it as a param to getTopPair

File: java/libs/src/test/java/org/physical_web/collection/PhysicalWebCollectionTest.java [hunk 2]
```diff
@@ -145,46 +129,4 @@ public void jsonDeserializeWorks() throws PhysicalWebCollectionException {
     assertEquals(pwsResult.getSiteUrl(), URL1);
     assertEquals(pwsResult.getGroupId(), GROUP_ID1);
   }
-
-  @Test
-  public void getPwPairsSortedByRankWorks() {
```
Comment (cco3) [inline java/libs/src/test/java/org/physical_web/collection/PhysicalWebCollectionTest.java]: This method still exists and needs to be tested, right?

**Comment by cco3** [comment]
LGTM.  Test the app to make sure it still works right and sorts by distance.

Please add a little more about why this change is desirable (lets client code provide relative sort comparators instead of demanding they provide an absolute rank.)

File: java/libs/src/test/java/org/physical_web/collection/PwPairTest.java [hunk 3]
```diff
@@ -40,7 +40,7 @@
   private static final double RANK2 = 0.9d;
   private UrlDevice mUrlDevice1;
   private PwsResult mPwsResult1;
-  private PwPair mPwPair1;
+  private PwPair mPwPair1 = null;
```
Comment (nondebug) [inline java/libs/src/test/java/org/physical_web/collection/PwPairTest.java]: This isn't needed as mPwPair1 will already be initialized to null.

**Pull request merged**